v2x_ros_driver/include/v2x_ros_driver/v2x_radio_client.h

```diff
@@ -110,13 +110,13 @@ class V2XRadioClient
     bool sendV2xMessage(const std::shared_ptr<std::vector<uint8_t>> &message);
 
     /**
-     * @brief Validate possible Message ID
+     * @brief Validate possible DSRCmsgID
```

Comment:
Let's update the name of this function from `IsValidMsgID()` to `isValidDSRCmsgID()`, if you agree?

Also consider changing the name of the input parameter from `msg_id` to `dsrc_msg_id`